extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -234,6 +257,27 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return null;
   }
 
+  /** @visibleForTesting */
+  divertExperiments() {
```

Comment:
could probably use a little more jsdoc or inline comments explaining what the experiments are